gateway/configuration.go

```diff
@@ -39,6 +39,8 @@ func loadConfig() {
 	diagnosisTickerTime = config.GetDuration("Diagnostics.gatewayTimePeriodInS", 60) * time.Second
 	// Enables accepting requests without user id and anonymous id. This is added to prevent client 4xx retries.
 	allowReqsWithoutUserIDAndAnonymousID = config.GetBool("Gateway.allowReqsWithoutUserIDAndAnonymousID", false)
+	// Sets the sampling rate for request latency stat
+	latencyStatSamplingRate = config.GetFloat64("Gateway.requestLatencyStatSamplingRate", 0.2)
```

Comment:
I feel it's better to keep the sampling configuration in stats module and stats module will expose a new function NewSampledTaggedStat with same parameters as NewTaggedStat. Essentially, we will just ask stats module to create a sampled stat and sampling is enforced inside stats module. I don't think we need different sampling rates in different modules.